src/stories/containers/TransparencyReport/components/AccountsSnapshot/components/Transaction/Transaction.tsx

```diff
@@ -1,12 +1,60 @@
+import styled from '@emotion/styled';
 import { useMediaQuery } from '@mui/material';
 import lightTheme from '@ses/styles/theme/light';
 import React from 'react';
 import MobileTransaction from './MobileTransaction';
+import TransactionAmount from './segments/TransactionAmount';
+import TransactionCounterParty from './segments/TransactionCounterParty';
+import TransactionHeader from './segments/TransactionHeader';
 
-const Transaction: React.FC = () => {
+interface TransactionProps {
+  name: string;
+  date: string;
+  toDate?: string | null;
+  txHash: string;
+  counterPartyName: string;
+  counterPartyAddress: string;
+  amount: number;
+  isIncomingTransaction?: boolean;
+}
+
+const Transaction: React.FC<TransactionProps> = ({
+  name,
+  date,
+  toDate,
+  txHash,
+  counterPartyName,
+  counterPartyAddress,
+  amount,
+  isIncomingTransaction = true,
+}) => {
   const isMobile = useMediaQuery(lightTheme.breakpoints.down('table_834'));
 
-  return isMobile ? <MobileTransaction /> : <div>Transaction</div>;
+  return isMobile ? (
```

Comment:
A question here.it's possible to have this approach with CSS?

Reply:
Yes, it is possible, but in this case, as those two components are very complex and have a lot of events and behavior I wanted to remove them entirely from the DOM instead of just hiding it to avoid performance issues or unexpected behavior